docs/tools/vdb_table/data/neo4j.json

```diff
@@ -123,30 +124,30 @@
     "comment": ""
   },
   "disk_index": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
     "comment": ""
```

Comment:
A description or source URL showing that the index is held on disk (without being in memory) would be required here. To my knowledge, Neo4j's vector index is held in memory (even though the vectors themselves are on disk).  Other vendors that have this option checked are pointing to docs that show usage of DiskANN, SAI indexes, and so on, so this should be marked "none" imo, unless there's a documentation page (or code) that you could link to showing the on-disk index with a description.